javascript/core/src/verbs/merge.ts

```diff
@@ -26,13 +26,33 @@ export interface MergeArgs extends InputColumnListArgs, OutputColumnArgs {
 	 * If it is not supplied, the values are just mashed together.
 	 */
 	delimiter?: string
+	unhot?: boolean
+	prefix?: string
+	keepOriginalColumns?: boolean
 }
 
 export const mergeStep: ColumnTableStep<MergeArgs> = (
 	input,
-	{ columns = [], strategy, to, delimiter = '' },
+	{
+		columns = [],
+		strategy,
+		to,
+		delimiter = '',
+		unhot = false,
+		prefix = '',
+		keepOriginalColumns = false,
+	},
 ) => {
-	const isSameDataTypeFlag: boolean = isSameDataType(input, columns)
+	const copyTable: ColumnTable = fromJSON(input.toJSON())
```

Comment:
I don't think you will need to use this copyTable line if you make unhotOperation a pure function and don't mutate the input table